jupyterhub_traefik_proxy/etcd.py

```diff
@@ -75,8 +79,22 @@ def _default_client(self):
                     self.etcd_traefik_prefix + "entrypoints/http/address",
                     ":" + str(urlparse(self.public_url).port),
                 ),
-                KV.put.txn(self.etcd_traefik_prefix + "api/dashboard", "false"),
-                KV.put.txn(self.etcd_traefik_prefix + "api/entrypoint", "http"),
+                KV.put.txn(
+                    self.etcd_traefik_prefix + "entrypoints/auth_api/address",
+                    ":" + self.traefik_auth_api_port,
```

Comment:
Here is where we might use `urlparse(self.traefik_api_url).netloc` which would be `127.0.0.1:8099` for the default.